.github/workflows/deployment-template.yml

```diff
@@ -51,14 +59,63 @@ jobs:
           OPENID_LOGOUT_URL: ${{ secrets.openid_logout_url }}
           GA_TRACKING_ID: ${{ secrets.ga-tracking-id }}
 
-      - name: Step 2 - Configure AWS Credentials
+      - name: Step 2 - Replace Tokens for logstash configuration
+        if: ${{inputs.deploy-env == 'preview' }}
+        uses: cschleiden/replace-tokens@v1
+        with:
+          files: '["${{ github.workspace }}/config/playground/metrics/logstash/${{inputs.deploy-env}}/logstash.yaml"]'
+          tokenPrefix: '${'
+          tokenSuffix: '}'
+        env:
+          OSD_USER: ${{ secrets.osd_user }}
+          OSD_USER_PASSWORD: ${{ secrets.osd_user_password }}
+
+      - name: Step 3 - Replace Tokens for tracing configuration
+        if: ${{inputs.deploy-env == 'preview' }}
```

Comment:
nit: add one space on the left as well `if: ${{ inputs.deploy-env == 'preview' }}`